src/osd/OSD.h

```diff
@@ -1444,7 +1452,7 @@ class OSD : public Dispatcher,
   void _remove_heartbeat_peer(int p);
   bool heartbeat_reset(Connection *con);
   void maybe_update_heartbeat_peers();
-  void reset_heartbeat_peers();
+  void reset_heartbeat_peers(bool all = true);
```

Comment:
can we drop the arg default, so it's clear which behavior the other caller(s) are getting?

Reply:
updated